Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -521,6 +536,8 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
     _handleFontVariationValueChanged(event)
     {
+        this._skipNextUpdate = true;
```

Comment:
The goal is to prevent DOM thrashing in the Fonts sidebar panel while dragging a slider. 
Noticeable by a jittery slider thumb. No need to update it with the value it itself originated. 
While dragging, the value from the model is always slightly behind the current value of the slider.

Reply:
Won't the throttling for updates already reduce DOM thrashing? I'm not sure its worth missing incoming updates that may not have originated from us to save an operation every quarter of a second (or every 10th/20th of a second if we reduce the intervals)

Reply:
> Won't the throttling for updates already reduce DOM thrashing?

Not quite. Even with throttling you can get into the scenario of updating the value for the slider you're currently dragging. It just happens less frequently. While dragging, the reaction to change is always for an older value, hence causing jitter. Not the end of the world, but it's there.